/**: # wildcard for the node name
  ros__parameters:
    moving_average_beta: [1.0, 1.0, 1.0]      # to disable set 1.0
    sigma_shutoff_value: [0.02, 0.02, 0.02]   # Values at which the uncertanty gets too great -> no thrust beyond this value
    i_controller_shutoff_error: [0.05, 0.05, 0.05]  # set to 0.0 to disable
    gains:
      p: [0.7, 0.7, 0.7]  # P gain for x1, x2 & x3
      i: [0.3, 0.3, 0.3]  # I gain for x1, x2 & x3
      d: [0.0, 0.0, 0.0]  # D gain for x1, x2 & x3